### PR TITLE
NG2-SANDBOX-264: redo of sandbox to support ng2

### PR DESCRIPTION
**Pull request by glawson**

### CODE REVIEW

File: dist/components/calendar.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+System.register(['angular2/core'], function(exports_1, context_1) {
```
Comment (tabersays) [inline dist/components/calendar.js]: Did you mean to push the compiled js of the angular2 calendar?

**Comment by tabersays** [comment]
@glawson You have a lot of intermediary files, do you really want to commit those files too or can we just let the ts transpiler make those?

File: dist/components/calendar.js (hunk 1 above)
Comment (glawson) [inline dist/components/calendar.js]: Yes - it's a distribution

**Comment by glawson** [comment]
@tabersays - in the dist, yes.  In the site too - but I'll remove in the next pr.

**Comment by tabersays** [comment]
@glawson I'll go ahead and merge it then.

**Pull request merged**